src/celeritas/io/ImportPhysicsVector.hh

```diff
@@ -60,7 +60,7 @@ struct ImportPhysics2DVector
 
     explicit operator bool() const
     {
-        return !x.empty() && !y.empty() && value.size() == x.size() * y.size();
+        return !value.empty() && value.size() == x.size() * y.size();
```

Comment:
So an empty vector is "true" because it's consistently defined, even though it's empty? The semantics of `operator bool` may be a little inconsistent across all our import data, we should at least mention in the documentation block that it's true if it's consistent regardless of being empty.

Reply:
This should still only be true for non-empty, right?

Reply:
Oh silly me, didn't read it thoroughly enough 🤦‍♂️